### ISSUE TITLE
[BUG] Tensorboard not logging any metrics for Autotrain on dgx

### ISSUE THREAD

**Issue by jmparejaz** [huggingface/autotrain-advanced#805]
### Prerequisites

- [X] I have read the [documentation](https://hf.co/docs/autotrain).
- [X] I have checked other issues for similar problems.

### Backend

Hugging Face Space/Endpoints

### Interface Used

UI

### Error Logs

![image](https://github.com/user-attachments/assets/a2727117-aea8-4975-b4c0-95c1139480d6)

### Additional Information

I have run multiple times finetuning LLM-ORPO with autotrain dgx and it is not logging to tensorboard the metrics
Deciding which hyperparameters to tweak without logs or when to stop training without looking the performance of the loss is not technically good. It would be amazing to have logging again.
I noticed that using the parameter "report" with WANDB doesnt work either. Any advice to make this work?
